src/OpenTelemetry.Instrumentation.Http/Implementation/HttpTagHelper.cs

```diff
@@ -94,6 +94,13 @@ public static string GetHostTagValueFromRequestUri(Uri requestUri)
             return hostTagValue;
         }
 
+        /// <summary>
+        /// Gets the OpenTelemetry standard uri tag value for a span based on its request <see cref="Uri"/>.
+        /// </summary>
+        /// <param name="uri"><see cref="Uri"/>.</param>
+        /// <returns>Span uri value.</returns>
+        public static string GetUriTagValueFromRequestUri(Uri uri) => string.Concat(uri.Scheme, Uri.SchemeDelimiter, uri.Authority, uri.PathAndQuery, uri.Fragment);
```

Comment:
do we have a way to avoid the string concats, if there is no username/password in the Uri?
if (uri has UsernameInfo)
{
 do what is done in this PR.
}
else
{
existing behavior.
}

Reply:
@cijothomas 
What do you think of something like
```
if string.IsNullOrEmpty(uri.UserInfo)
{
    return uri.OriginalString;
}
return string.Concat(uri.Scheme, Uri.SchemeDelimiter, uri.Authority, uri.PathAndQuery, uri.Fragment);
```
?

Reply:
Addressed https://github.com/open-telemetry/opentelemetry-dotnet/pull/1961/commits/91d083d953c96e935826cab2cc03b7c875120d0c